### ISSUE TITLE
Software Design: Sequence Diagram - Edit Profile

### ISSUE THREAD

**Issue by codingAku** [bounswe/bounswe2022group2#120]
### Issue Description

This is a sub-issue of #107 .
As discussed in our meeting, I have the responsibility of completing the Edit Profile Sequence Diagram. The diagram can be accessed from [Lucidchart](https://lucid.app/documents/view/cdbe93f5-ca31-44f1-bb57-5eef9dbf11ee).

### Final Actions

After the diagram is completed, we will put them all in one page.

### Deadline of the Issue

6.04.2022 - 16:00

### Reviewer

Hasan Can Erol

### Deadline for the Review

6.04.2022 - 17.00

**Comment by codingAku** [comment]
UPDATE: Initial version of the editing profile diagram has been created using the Class diagram of Profile and for now it looks like this. Enhancements will be made after the meeting for Sequence Diagrams and PS Session.
![sequc](https://user-images.githubusercontent.com/64018144/161970323-de639771-c696-41c9-bf1d-7411436bcdae.jpg)

**Comment by codingAku** [comment]
![classlast](https://user-images.githubusercontent.com/64018144/163045461-835cc8d9-9a5e-4bf6-b69c-b72dadd51602.jpg)
![use case](https://user-images.githubusercontent.com/64018144/163045475-9de810d9-d861-4ded-b5c6-229a238a2f3f.jpg)
According to class and use case diagrams, I have added achievements and interest areas methods. Additionally, I removed get methods and left only set methods for editing. I have changed lifecycle blocks also. 

Final version of sequence diagram of Profile is down below:
![latestsq](https://user-images.githubusercontent.com/64018144/163046789-f8d4006f-2947-494e-852f-c87a0d50f4d8.jpg)
